test/data/brave_ads/component/rich_media/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "Brave NTP sponsored media component",
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0EPcM/eCBXicL5eOJwfAAKsGXD0uSrQzxwl0Qc1wp7DbQUtyd3IW3w/5AXi/HyfSVSw1YNMj+1732TsMA5u2u4IYhgE9TSens7Tk3WGNop/El7ZPLKdHk93CDCYNdyGPEAeXWXJS5Q0wPkOnB/3I258JlI9r3XUE+AZY5aXLtl8DcCVLwVhg/NG7cPqM7TN3TXAJ5x69mPcL9xyhzxiGOWPk/dVGf1xilRQG/f6JkT3d1ikkvsImu+GRe5pJ3slzUTO+uAe9ddfCTsEg8+DlAgBo0/CNXCzPn7QVwNvUbVKTnnBHh48H27Xk5gs6jI4jeMhCmx5VhgS5f/T0LNL6KQIDAQAB",
+  "manifest_version": 2,
+  "name": "Brave NTP sponsored media",
+  "version": "1.0.999"
+}
```

Comment:
@aseren lets remove `manifest.json` (this file) because it is not needed for tests.

Reply:
Fixed